rvs/conf/nv48/gpup_single.conf

```diff
@@ -0,0 +1,174 @@
+# ################################################################################
+# #
+# # Copyright (c) 2018-2022 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
Update year to 2024 in all the new conf. files

Reply:
done